apps/src/lab2/views/components/Instructions/ResourcePanel/index.tsx

```diff
@@ -441,6 +441,7 @@ const ResourcePanel: React.FC<ResourcePanelProps> = ({
             id={resourcePanelLinksElementId}
             className={classNames(styles.bottomTabs)}
           >
+            <ResourcePanelExtraLinks levelId={levelId} theme={theme} />
```

Comment:
Do we want the collapse/expand button to also be small?

Reply:
I think this one is supposed to be bigger so it's good as-is.